wagtail/snippets/templates/wagtailsnippets/bulk_actions/confirm_bulk_delete.html

```diff
@@ -0,0 +1,60 @@
+{% extends 'wagtailadmin/bulk_actions/confirmation/base.html' %}
+{% load i18n wagtailusers_tags wagtailadmin_tags %}
+
+{% block titletag %}
+    {% if items|length == 1 %}
+        {% blocktrans trimmed with snippet_type_name=model_opts.verbose_name %}Delete {{ snippet_type_name }}{% endblocktrans %} - {{ items|first }}
+    {% else %}
+        {{ items|length }} {{ model_opts.verbose_name_plural|capfirst }}
+    {% endif %}
+{% endblock %}
+
+{% block header %}
+    {% trans "Delete " as delete_str %}
+    {% if items|length == 1 %}
+        {% include "wagtailadmin/shared/header.html" with title=delete_str subtitle=items.0.item icon="snippet" only %}
+    {% else %}
+        {% include "wagtailadmin/shared/header.html" with title=delete_str subtitle=model_opts.verbose_name_plural|capfirst icon="snippet" only %}
+    {% endif %}
+{% endblock header %}
+
+{% block items_with_access %}
+    {% if items %}
+        {% if items|length == 1 %}
+            {% usage_count_enabled as uc_enabled %}
+            {% if uc_enabled %}
+                <div class="usagecount">
+                    <a href="{{ items.0.usage_url }}">{% blocktrans trimmed count usage_count=items.0.get_usage.count %}Used {{ usage_count }} time{% plural %}Used {{ usage_count }} times{% endblocktrans %}</a>
```

Comment:
I haven't verified why (as other bulk action templates seem to work just fine with the same way), but this fails because `items.0` is not the snippet instance, the snippet instance is `items.0.item`.

```suggestion
                    <a href="{{ items.0.item.usage_url }}">{% blocktrans trimmed count usage_count=items.0.item.get_usage.count %}Used {{ usage_count }} time{% plural %}Used {{ usage_count }} times{% endblocktrans %}</a>
```

You can verify this by adding:

```py
WAGTAIL_USAGE_COUNT_ENABLED = True
```

and try deleting a single snippet using the bulk actions button.

Reply:
Yep, in case of snippets, the item is actually a dict of {item, edit_url}, so it should indeed be `item.item`. Nice catch !